core/src/test/java/com/google/zxing/qrcode/detector/AlignmentPatternFinderTestCase.java

```diff
@@ -0,0 +1,140 @@
+package com.google.zxing.qrcode.detector;
+
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import com.google.zxing.NotFoundException;
+import com.google.zxing.ResultPointCallback;
+import com.google.zxing.common.BitMatrix;
+
+public class AlignmentPatternFinderTestCase extends Assert {
+	
+	private static BitMatrix matrix; 
+	private static ResultPointCallback resultPointCallback;
+	private static AlignmentPatternFinder APF; 
```

Comment:
Usually the variable should be cammelcase (like `apf`), but not sure if this applies in this case.